openqaoa/cost_function.py

```diff
@@ -38,47 +38,52 @@ def expectation_value_classical(counts: Dict, hamiltonian: Hamiltonian):
     return cost
 
 
-def cvar_expectation_value_classical(alpha: float, counts: Dict, hamiltonian: Hamiltonian):
-    """
-    CVaR computation of cost function. For the definition of the cost function, refer
-    to https://arxiv.org/abs/1907.04769.
+def cvar_expectation_value_classical(counts: Dict, hamiltonian: Hamiltonian, alpha: float):
+	"""
+	CVaR computation of cost function. For the definition of the cost function, refer
+	to https://arxiv.org/abs/1907.04769.
 
-    Parameters
-    ----------
-    alpha: `float`
-            The CVaR parameter.
-    counts: `dict`
-            The counts of the measurements.
-    hamiltonian: `Hamiltonian`
-            The Cost Hamiltonian defined for the optimization problem
-    """
-    assert alpha > 0 and alpha < 1, "Please specify a valid alpha value between 0 and 1"
-    shots = sum(counts.values())
+	Parameters
+	----------
+	counts: `dict`
+		The counts of the measurements.
+	hamiltonian: `Hamiltonian`
+		The Cost Hamiltonian defined for the optimization problem
+	alpha: `float`
+		The CVaR parameter.
+	"""
+	assert alpha > 0 and alpha < 1, "Please specify a valid alpha value between 0 and 1"
```

Comment:
I guess it should be <=1, no?

Reply:
hey, no, there is a separate function that executes for alpha=1 for better speed. We don't need to sort the counts for the case of alpha=1